schemas/2015-10-01/Microsoft.DataFactory.json

```diff
@@ -141,7 +141,7 @@
           ]
         },
         "type": {
-          "enum": [ "Microsoft.DataFactory/datafactories/pipelines" ]
+          "enum": [ "Microsoft.DataFactory/datafactories/datapipelines" ]

```

Comment:
@wenwu449 should be fixed now, if I understood you correctly (changed line 128 to be "datapipelines")